src/templates/static/legal/risk_disclaimer.jsx

```diff
@@ -2,59 +2,64 @@ import React from 'react';
+        <p>{it.L('Other Risks: There are other risks related to trading Investment Products and foreign currencies that cannot be described in detail in this document. Generally, however, securities, options, and currency transactions involve exposure to a combination of the following risk factors: market risk, credit risk, settlement risk, liquidity risk, operational risk, and legal risk. For example, there can be serious market disruptions if local or overseas economic, political, or other unforeseen events affect the market. In addition to these types of risk, there may be other factors such as accounting and tax treatment issues that the client should consider.')}</p>
 
-        <h2 data-anchor='market-opinions'>{it.L('Market Opinions')}</h2>
-        <p>{it.L('All opinions, news, research, analysis, prices or other information contained on this website are provided as general market commentary and do not constitute investment advice. [_1] will not accept liability for any loss or damage, including but without limitation to, any loss of profit that may arise directly or indirectly from use of or reliance on such information.', it.website_name)}</p>
+        <h2 data-anchor='market-opinions'>{it.L('Market opinions')}</h2>
+        <p>{it.L('All opinions, news, research, analysis, prices, or other information contained on this website are provided as general market commentary and do not constitute investment advice. [_1] will not accept liability for any loss or damage, including, but without limitation to, any loss of profit that may arise directly or indirectly from the use of, or reliance on, such information.', it.website_name)}</p>
 
-        <h2 data-anchor='internet-trading-risks'>{it.L('Internet Trading Risks')}</h2>
-        <p>{it.L('There are risks associated with utilizing an Internet-based deal execution trading system, including but not limited to the failure of hardware, software and Internet connection. Since [_1] does not control signal power, its reception or routing via Internet, configuration of your equipment or reliability of its connection, we cannot be responsible for communication failures, distortions or delays when trading via the Internet.', it.website_name)}</p>
+        <h2 data-anchor='internet-trading-risks'>{it.L('Internet trading risks')}</h2>
+        <p>{it.L('There are risks associated with utilising an Internet-based deal execution trading system, including, but not limited to, the failure of hardware, software and Internet connection. Since [_1] does not control signal power, its reception or routing via Internet, the configuration of the client\'s equipment, or the reliability of its connection, the Company cannot be responsible for communication failures, distortions, or delays during online trading.', it.website_name)}</p>
```

Comment:
```suggestion
        <p>{it.L('There are risks associated with utilising an Internet-based deal execution trading system, including, but not limited to, the failure of hardware, software, and Internet connection. Since [_1] does not control signal power, its reception or routing via Internet, the configuration of the client\'s equipment, or the reliability of its connection, the Company cannot be responsible for communication failures, distortions, or delays during online trading.', it.website_name)}</p>
```